doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/3_2_0.xhtml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<link rel="stylesheet" type="text/css" href="new.css" />
+<title>What's new in hale studio 3.2?</title>
+</head>
+<body>
+	<h1>What's new in hale studio 3.2?</h1>
+
+	<ul>
+		<li>Read interpolated Arc geometries from GML</li>
+		<li>Support for multiple instance validators</li>
+		<li>New <em>Project validator</em></li>
+		<li>Request pagination for WFS sources</li>
+		<li>Enhanced UI support for project variables</li>	
+	</ul>
+
+	<p>
+		For a full list of changes, please check out the <a
+			href="https://github.com/halestudio/hale/blob/master/CHANGELOG.md#320"
+			target="_blank">changelog on GitHub</a>
+	</p>
+
+	<p>The majority of the new features and improvements were made
+		possible through our contracts with more than 20 customers. We'd like
+		to thank you all for your support!</p>
+
+	<table border="0" cellpadding="15" cellspacing="0" class="news" id="3_2_0">
+		<tbody>
+			
+			<!-- New features -->
+			<tr>
+				<td class="section" colspan="2" id="3_2_0_Features" rowspan="1"><h2>New features</h2></td>
+			</tr>
+
+			<tr id="arcs">
+				<td class="title">Arc geometries</td>
+				<td class="content">
+					<p>
+						Regarding support for geometries in GML there are still some
+						geometry types not supported by hale studio. To reduce that list
```

Comment:
Until now, hale didn't support geometries with non-linear interpolation between points, such as arcs. We now added a robust interpolation method that processes Arcs, ... and CircleByCenterPoints and creates regular LineStrings and Polygons.